yolo_model:

  config_file:
    name: tiny-yolo.cfg
  weight_file:
    name: down.backup
  threshold:
    value: 0.5
  detection_classes:
    names:
      - roomba